common/django_urls.py

```diff
@@ -1,29 +1,40 @@
-from django.conf import settings
 import urllib.parse
 
-class UrlMixin(object):  # pragma: no cover
-    
+from django.conf import settings
+
+
+class UrlMixin(object):
+    """
+    DEPRECATED - DO NOT USE
+
+    This is kept here for legacy reasons as migrations require it and in case
+    we need to reffer to it. It was used in the Talk and Speaker models.
+    """
```

Comment:
Removed this from `Talk` and `Speaker` as it is a 11y old mixin that does not seem to be used, nor these methods seem to exist in django any more. Leaving this file in for reference and in case we need to revert.